products/health-care/engineering/documenation/audit.md

```diff
@@ -0,0 +1,61 @@
+# Documentation audit Scope of Work
+
+The primary objective is to produce standard, long living documentation for the following APIs:
```

Comment:
should we say something like: 'long living documentation for all APIs, including the following:... '